libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKOAuth2.h

```diff
@@ -78,6 +78,13 @@ typedef NS_ENUM(NSInteger, SFLogoutReason) {
     SFLogoutReasonRefreshTokenRotated // "Refresh token rotated"
 };
 
+typedef NS_ENUM(NSInteger, SFLoginReason) {
+    SFLoginReasonUnknown,
```

Comment:
these are tbd, I was thinking we should also have something for AuthHelper entry point since that's only called by apps